spec/rubocop/cop/style/if_with_semicolon_spec.rb

```diff
@@ -28,4 +28,57 @@ class Hash
       end if RUBY_VERSION < "1.8.7"
     RUBY
   end
+
+  context 'when elsif is present' do
+    it 'accepts without `else` branch' do
+      expect_offense(<<~RUBY)
+        if cond; run elsif cond2; run2 end
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Do not use if x; Use the ternary operator instead.
+      RUBY
+
+      expect_correction(<<~RUBY)
+        if cond
+          run
+        elsif cond2
+          run2
+        end
+      RUBY
+    end
+
+    it 'accepts second elsif block' do
+      expect_offense(<<~RUBY)
+        if cond; run elsif cond2; run2 elsif cond3; run3 else dont end
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Do not use if x; Use the ternary operator instead.
```

Comment:
I think we should change the message here. Suggesting to convert `elsif` to ternary is not a good idea.

Reply:
True, thanks for the feedback